package.json

```diff
@@ -21,7 +21,8 @@
     "clsx": "^1.2.1",
     "prism-react-renderer": "^1.3.5",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "yarn": "^1.22.22"
```

Comment:
(non-blocking, not really a big deal) Hmmm not sure if this should be here, typically yarn is installed globally for most folks.